packages/cli/src/ui/privacy/PrivacyNotice.tsx

```
@@ -0,0 +1,42 @@
+}: {
+  config: Config;
+  onExit: () => void;
+}) => {
+  const authType = config.getContentGeneratorConfig()?.authType;
+
+  switch (authType) {
+    case AuthType.USE_GEMINI:
+      return <GeminiPrivacyNotice onExit={onExit} />;
+    case AuthType.USE_VERTEX_AI:
+      return <CloudPaidPrivacyNotice onExit={onExit} />;
+    case AuthType.LOGIN_WITH_GOOGLE_PERSONAL:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)